src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>An authentication exception occurred.</source>
+                <target>Autentiseringa har feila.</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>Authentication credentials could not be found.</source>
+                <target>Påloggingsinformasjonen vart ikkje funnen.</target>
+            </trans-unit>
+            <trans-unit id="3">
+                <source>Authentication request could not be processed due to a system problem.</source>
+                <target>Autentiseringa vart ikkje prosessert på grunn av ein systemfeil.</target>
```

Comment:
Above the more layman term `Pålogging` is used, should it be used here to?

Context:
- `Pålogging` is more or less same as `Login`
- `Authentication` in Norwegian (all forms), is less common then in english
  - That said I'm in tech sector surrounded by these words in english all the time, so I'm partial. Maybe the wording should have been simplified also in english, or maybe it's supposed to be technical and thus should also be in Norwegian 🙃

Reply:
The same goes for token/teikn. How technical should it be? I did translate that, but not "nonce". I'll check these against the list Språkrådet made for this purpose.

Reply:
I was checking the same thing and I think that is ok as is (didn't find anything that would be making sense on nonce on some quick looks)

Reply:
I think now it's best to replace all of these with innlogging. Slightly more layman, and referenced in the dictionary.